packages/block-library/src/heading/edit.native.js

```diff
@@ -58,7 +58,7 @@ class HeadingEdit extends Component {
 				<Toolbar controls={ range( 2, 5 ).map( this.createLevelControl.bind( this ) ) } />
 				<RichText
 					tagName={ tagName }
-					content={ { contentTree: attributes.content, eventCount: attributes.eventCount } }
+					content={ { contentTree: attributes.content } }
```

Comment:
You can pass `attributes.content` directly without the object wrapper.

Reply:
`RichText` expects to receive an object, since there is an optional key `forceUpdate` that the components can add to force the update/rendering of the underlying native component.